geps/gep-713/index.md

```diff
@@ -1,1157 +1,1082 @@
 
-### Why use Policy Attachment at all?
+The semantics of *applying rules to a target* characterizes these particular instances of metaresources as *policies*, thus here emerging in relation to this pattern the concept of *Policy Attachment*.
 
+Policies are therefore a strict subset of metaresources which distinguish itself subtly from the whole on the fact that the augmented behavior described by policies resemble *conditional rules*, while the mechanism of targeting an object or set of objects (“attaching to objects”) is nonetheless exactly the same as in the general definition of metaresources.
 
-Consistent UX across GW implementations
+Without loss of generality, the very activation of the context (targets) of metaresources can be understood as the _conditions_ for the enforcement of the intended augment behavior (rules) that are specified in the metaresources, and therefore these can often be refered to as policies interchangeably.
 
-Support for common tooling such as gwctl that can compute and display effective policies at each layer
+## Applications
 
-Avoid annotation hell
+### Reasons for using metaresources and policies
 
+These are a few reasons for using metaresources and policies over another (possibly more direct) way to modify the spec (“augment the behavior”) of an object:
 
-### Direct Policy Attachment
+* Extending otherwise stable APIs – e.g. to specify additional network settings for the Kubernetes Service object.
+* Defining implementation-specific functionalities for otherwise common APIs－e.g. to specify implementation-specific behavior for Gateway API HTTPRoute objects.
+* Decoupling concerns for targeting personas with specific functionality and configuration－ delegation of responsibilities, fine-grained RBAC.
+* Decoupling responsibility over the management and implementation of the metaresources themselves.
+* Avoid alternatives based on annotations which are often non-standardized, poorly documented, and generally hard to maintain.
 
-For more description of the details of Direct Policy Attachment,
-see [GEP-2648](https://gateway-api.sigs.k8s.io/geps/gep-2648/).
+### Examples of applications of metaresources and policies
 
-### Inherited Policy Attachment
+* Specifying traffic rules for Kubernetes Services
+* Configuring implementation-specific authentication features on ingress Gateways
+* Delegating responsibility over parts of the configuration (e.g. auth) to other users
+* Inspecting and auditing focused aspects of a resource (e.g. auth policies)
 
-For more description of the details of Inherited Policy Attachment,
-see [GEP-2649](https://gateway-api.sigs.k8s.io/geps/gep-2649/).
+## Guide-level explanation
```

Comment:
What's "Guide-level"?  Implementation guide?

Reply:
It's a mix of implementation and user's guide.
It's meant for all readers that want to understand how metaresources work.

Because many things related to metaresources are implementation-specific, it's hard to write a guide that says exactly how they should be implemented, so we hint on the different options, what's a SHOULD and what's a MUST within those options, lay out some abstract definitions that are generic enough for all implementations and kinds of metaresources, etc.

Arguably, this is also useful to users of metaresources, not only to the implementations－even if only to the most interested users who will care to learn about Gateway API at this level.

Reply:
Tried to make that clear in the _caput_ of the section.